parts/kubernetesagentcustomdata.yml

```diff
@@ -14,7 +14,7 @@ write_files:
   content: |
     [Service]
     ExecStart=
-    ExecStart=/usr/bin/docker daemon -H fd:// --storage-driver=overlay
+    ExecStart=/usr/bin/docker daemon -H fd:// --storage-driver=overlay --log-driver=journald --bip={{WrapAsVariable "dockerBridgeCidr"}}
```

Comment:
@dmccaffery Jack's spot on. We can't use journald as a log driver. Overall this PR looks good. I'm checking on some CI coverage we can add as well.